beta-src/src/state/game/game-api-slice.ts

```diff
@@ -157,10 +159,12 @@ const gameApiSlice = createSlice({
       // fetchGameOverview
       .addCase(fetchGameOverview.pending, (state) => {
         state.apiStatus = "loading";
+        state.activity.makeNewCall = false;
```

Comment:
Isn't this already set to false since you set this in initial-state

Reply:
initially yes, but game overview is the poll and this runs often based on user activity, if I remove this there will be moments where the API call is pending but makeNewCall is set to `true`, at which point it will attempt to make a new API call while the previous one is in progress.